runtime/marshal_jsonpb.go

```diff
@@ -67,6 +67,12 @@ func (j *JSONPb) marshalNonProtoField(v interface{}) ([]byte, error) {
 		rv = rv.Elem()
 	}
 
+	if rv.Kind() == reflect.Slice {
+		if rv.IsNil() && j.EmitDefaults {
+			return []byte("[]"), nil
+		}
```

Comment:
I suppose we could put this all in one big if instead to reduce indentation?

Reply:
That works for me, honestly wasn't sure if you wanted me to return `"null"` if j.EmitDefaults wasn't set.  I can make it all one big if.

👍